test/Makefile

```diff
@@ -0,0 +1,14 @@
+TEST=1
+OFILES := $(OFILES) ../src/infoPanel/imagesCache.o
+include ../src/common/config.mk
+
+TARGET = test_infoPanel
+LDFLAGS := $(LDFLAGS) -lSDL_image -lSDL -lgtest -lgtest_main -lpthread
+
+include ../src/common/commands.mk
+include ../src/common/recipes.mk
+
+../src/infoPanel/imagesCache.o: ../src/infoPanel/imagesCache.c
```

Comment:
@Aemiii91 can we find a way to handle this by recipes.mk instead automatically (%.o target is not working for me)

Reply:
Take a look at how I'm doing it here: https://github.com/OnionUI/Onion/blob/new-build-structure/src/gameSwitcher/Makefile